src/style.css

```diff
@@ -0,0 +1,46 @@
+.watch-container {
+  position: absolute;
+  left: 50%;
```

Comment:
no need to mention both left and right, something one is enough

Reply:
Thanks for review, I corrected mistakes!